qna/src/app/parser/parser.py

```diff
@@ -82,20 +109,46 @@ def getPostsFromThreads(threads):
                                                          'X-img': msg['X-img'],
                                                          'Text': msg._payload,
                                                          })
-    with open('app/storage/questions2017_UE.json', 'w') as outfile:
+    with open('app/storage/questions2017_Universal_Encoder.json', 'w') as outfile:
         json.dump(questions, outfile, indent=4)
     print("Finished loading Json...")
 
 
-def get_vectors(question):
-    '''
-    Vectorises the subject-line of the given question.
+def get_summarisation(data, tokenizer, model):
+    input = tokenizer.encode(data, return_tensors="pt",
+                             max_length=512, truncation=True)
+    # generate the summarization output
+    outputs = model.generate(
+        input,
+        max_length=50,
+        min_length=30,
+        length_penalty=2.0,
+        num_beams=4,
+        early_stopping=True)
+    # just for debugging
+    return tokenizer.decode(outputs[0])
+
 
-    :param question: Element of question dictionary
-    :return vec: Sentence vector for question
+def preprocess(data):
     '''
-    module_url = "https://tfhub.dev/google/universal-sentence-encoder/4"
-    model = hub.load(module_url)
-    vec = model([question])[0]
-    vec = tf.reshape(vec, (-1, 1))
-    return vec
\ No newline at end of file
+    # I still don't know if I will use htis now.
+    '''
```

Comment:
It seems that this is used in the basic CLI making this comment misleading, probably should be removed?

Reply:
Yeah removing